### DISCUSSION TITLE
Updating the app

### DISCUSSION THREAD

**Discussion by mrozj** [JonathanTreffler/backblaze-personal-wine-container#33] [General]
I am sure that I am missing something simple here but how do we keep the Backblaze app itself updated. I am running this on Unraid and have forced the update but that app is still several versions behind.

**Comment by sisaacrussell** [comment]
See discussion here: [https://github.com/JonathanTreffler/backblaze-personal-wine-container/issues/5](https://github.com/JonathanTreffler/backblaze-personal-wine-container/issues/5)

**Comment by devanteweary** [comment]
Once I try to run that wine64 command, I get this: 
'/config/wine' is not owned by you

What about you guys?

**Reply by traktuner** [reply]
you need to perform those commands as the user "app", not as "root" since app is the user who owns the wineprefix folder.

**Comment by JonathanTreffler** [comment]
Not relevant since v1.6 with auto update functionality, closing :)
